### PR TITLE
Use govuk_app_config & GovukError

### PR DESCRIPTION
**Pull request by barrucadu**

### CODE REVIEW

File: config/application.rb [hunk 1]
```diff
@@ -19,10 +19,6 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Raven.configure do |config|
-  config.dsn = ENV["SENTRY_DSN"]
```
Comment (huwd) [inline config/application.rb:23]: Interesting so where does the DSN come from now?
Do we end up using a GOV.UK common one?

Will we need to change any ENVVars with this?

Wondering if there's any assumption the var will just be there via pupped but we'll have to explicitly put it in?

Comment (barrucadu) [reply config/application.rb:23]: Both our apps(1) and normal GOV.UK apps(2) use the same env var, `SENTRY_DSN`, so it should just work.

(1) https://github.com/alphagov/govuk-attribute-service-prototype/blob/main/concourse/tasks/deploy-to-govuk-paas.yml#L42
(2) https://github.com/alphagov/govuk-puppet/blob/master/modules/govuk/manifests/app/config.pp#L200-L202

Comment (barrucadu) [reply config/application.rb:23]: The `SENTRY_CURRENT_ENV` var we set is also right: https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config/govuk_error/configuration.rb#L11

Comment (huwd) [reply config/application.rb:23]: ace, thanks for confirming!

**Pull request merged**